src/tests/store/activities/reducer.test.js

```diff
@@ -0,0 +1,83 @@
+import activitiesReducer from '../../../store/activities/reducer';
+import testState from '../../fixtures/state';
+import activity from '../../fixtures/activity';
+import {
+  ADD_ACTIVITY,
+  REMOVE_ACTIVITY,
+  EDIT_ACTIVITY,
+} from '../../../store/activities/actionTypes';
+
+test('should set default activities state', () => {
+  const initAction = { type: '@@INIT' };
```

Comment:
we don't have to test init action, as this is covered 1000 times better by lib authors :P